### DISCUSSION TITLE
Adding INKBIRD IAM-T1 Air Quality Monitor

### DISCUSSION THREAD

**Discussion by kiselio** [theengs/decoder#414] [New devices/things integration]
Hello,

I opened a pull request for adding the INKBIRD IAM-T1 air quality monitor, a Bluetooth device which requires an active connection for polling.

https://github.com/theengs/decoder/issues/413

I just wanted to start a discussion and maybe get someone who understands services and characteristics better to help out and integrated this device.

let me know what is needed, I can provide it.

**Comment by DigiH** [comment]
See my comment in the issue you opened.

If you could post some advertisement data the INKBIRD IAM-T1 might be broadcasting it will help to see if Theengs Decoder will be able to add a decoder for it.
